### DISCUSSION TITLE
Liquity Chicken Bonds

### DISCUSSION THREAD

**Discussion by 0x4007** [pavlovcik/uad-ui#207] [Ideas]
I'm a huge fan of Liquity and looks like they figured a bespoke bonds mechanism. I'm very inclined to study their approach and potentially use it in place of Olympus bonds. 

https://twitter.com/liquityprotocol/status/1546877322564968449?s=21&t=gePN5gtCZli2korMogLDHg

**Comment by sergfeldman** [comment]
[whitepaper_chicken_bonds_v0.1.pdf](https://github.com/ubiquity/ubiquity-dollar-development/files/9102437/whitepaper_chicken_bonds_v0.1.pdf)

**Comment by sergfeldman** [comment]
Explanation https://gist.github.com/peteriscan/a1ebde83ed7dfa1a3ffca5603efae8e4

**Comment by sergfeldman** [comment]
![LUSD_Chicken_Bonds_-_User_Overview](https://user-images.githubusercontent.com/15567379/178977340-457d0dce-3ce1-41bf-80e6-a04ef206f4b2.png)
![treasury-buckets](https://user-images.githubusercontent.com/15567379/178977350-2a5d823e-7ebe-489a-b8e9-6fc958b02866.png)

**Comment by 0x4007** [comment]
Voke from Concave suggests that we deploy two bonds instances. Dio on their team will set up a Concave style bonds protocol for us. However their implementation has an off-chain and proprietary blackbox dependency controlled by them which is the original reason why I was bearish on their implementation. However, if we have a fully on-chain Chicken Bonds implementation to fall back on then we'll be okay. 

As a final note, Voke made a compelling case for why we should also aggressively pursue a Curve gauge in tandem with our bonds deployment. He said it could be a great (temporary) revenue source.
